content/en/docs/concepts/overview/components.md

```diff
@@ -23,14 +23,40 @@ Here's the diagram of a Kubernetes cluster with all the components tied together
 {{% /capture %}}
 
 {{% capture body %}}
-## Control Plane Components
+## Node components
 
-The Control Plane's components make global decisions about the cluster (for example, scheduling), as well as detecting and responding to cluster events (for example, starting up a new {{< glossary_tooltip text="pod" term_id="pod">}} when a deployment's `replicas` field is unsatisfied).
+In Kubernetes, your application runs inside a group of containers called
+a {{< glossary_tooltip text="_pod_" term_id="pod" >}}. The idea of a pod
+is to isolate different workloads from each other. The computers that run
+pods are called _nodes_, and one node can run pods from many different
+workloads.
```

Comment:
Thank you, this is good. Might reword just a little:

```
In Kubernetes, your application's containers are grouped together in
a {{< glossary_tooltip text="_pod_" term_id="pod" >}}. Pods are an abstraction
that simplifies application management, resource sharing, and communication.
The computers, that run pods are called _nodes_, and one node can run pods
from many different workloads.
```

Add a link to this: https://kubernetes.io/docs/concepts/workloads/pods/pod/#motivation-for-pods as well, very useful.

Main change, I think, is to clarify the purpose of pods and align it with the content in the above link.

Reply:
>  Pods are an abstraction that simplifies application management, resource sharing, and communication.

I'm worried that wording like that, especially “abstraction”, might scare some readers off. I don't want the tone to sound like an academic paper.

I'll see if I can find the best of both.

Reply:
That's a good point. That said, we already refer to pods elsewhere in the docs (specifically in the references page) as a "higher-level abstraction". So I wanted to be internally consistent. Anyone scared off by terminology is going to be scared off by Kubernetes very quickly. :)

That said, we certainly can just say "Pods simplify application management..." 
The "are an abstraction" is just unnecessary sugar glaze for sure at this stage.

Reply:
A pod is a group of containers. I think that is fine.

Does the glossary link to the pod documentation?
https://kubernetes.io/docs/concepts/workloads/pods/pod/
https://kubernetes.io/docs/concepts/workloads/pods/pod-overview/

Also, it seems like those 2 pages could be merged or refactored.

Reply:
> https://kubernetes.io/docs/concepts/workloads/pods/pod/
> https://kubernetes.io/docs/concepts/workloads/pods/pod-overview/
> 
> Also, it seems like those 2 pages could be merged or refactored.

I like to keep PRs small (and reviewable), so in my view it's better for _this_ PR not to tackle that.

@bgrant0607 also asked _Does the glossary link to the pod documentation?_. Glossary tooltips renders as hyperlinks; usually to the relevant concept page.

Reply:
Thanks.

Is this PR still WIP?

Reply:
I've rebased it, should be good to merge I think.